python-frontend/typeshed_serializer/serializer/serializers.py

```diff
@@ -149,27 +173,39 @@ def build_for_every_python_version(self):
         model_by_version: dict[str, dict[str, ModuleSymbol]] = {}
         for major, minor in SUPPORTED_PYTHON_VERSIONS:
             opt = get_options((major, minor))
+            self.logger.info(f"Building for python version {major}.{minor}")
             build_result, source_paths = self.get_build_result(opt=opt)
             modules = {}
             for file in build_result.files:
                 path = build_result.files[file].path
-                if self.is_third_parties and (path not in source_paths or any(
-                        excluded_package in path for excluded_package in self.EXCLUDED_PACKAGES)):
+                if self.is_third_parties and (
+                    path not in source_paths
+                    or any(
+                        excluded_package in path
+                        for excluded_package in self.EXCLUDED_PACKAGES
+                    )
+                ):
                     # build_result contains more modules from stdlib unrelated to third_parties
                     continue
+
                 ms = ModuleSymbol(build_result.files.get(file))
                 modules[ms.fullname] = ms
             model_by_version[f"{major}{minor}"] = modules
         return model_by_version
 
     def get_build_result(self, opt=get_options()):
-        build_result, source_paths = walk_typeshed_third_parties(opt) if self.is_third_parties \
+        build_result, source_paths = (
+            walk_typeshed_third_parties(opt)
+            if self.is_third_parties
             else walk_typeshed_stdlib(opt)
+        )
         return build_result, source_paths
 
     def is_exception(self, file, build_result, source_paths):
         file_path = build_result.files[file].path
-        return self.is_third_parties and file_path not in source_paths
+        return (
+            self.is_third_parties and file_path not in source_paths
+        ) or "/numpy/" in file_path
```

Comment:
Is that intentional or a remnant ?

Reply:
No good catch this is an error during merge!